platform/golden/bullseye.pkr.hcl

```diff
@@ -97,7 +97,7 @@ source "virtualbox-iso" "base-debian-amd64" {
   nic_type             = "virtio"
   hard_drive_interface = "virtio"
   rtc_time_base        = "UTC"
-  shutdown_command     = "echo '${var.ssh_password}' | sudo -S shutdown -P now"
+  shutdown_command     = "echo '${var.ssh_password}' | sudo -S /usr/sbin/shutdown -P now"
```

Comment:
`/usr/sbin` not in the root path? Doesn't matter for the purposes of merging the PR, just seems odd. :)

Reply:
@rstarmer can you respond on these questions? I am trying to learn along the way (this feels wrong, which means `/etc/environment` may not be right).

Reply:
This was to resolve an issue with running shutdown, and I'm not sure why the environment worked initially, and then didn't.  I'll do a bit more debugging, and I agree, I've not seen a single example where the full path to shutdown was required.